src/__experimental__/components/radio-button/radio-button-group.spec.js

```diff
@@ -8,17 +8,30 @@ import { StyledRadioButtonGroup } from './radio-button.style';
 import Label from '../label';
 import LabelStyle from '../label/label.style';
 import { assertStyleMatch } from '../../../__spec_helper__/test-utils';
+import ValidationIconStyle from '../../../components/validations/validation-icon.style';
```

Comment:
i think the naming rule is `StyledXxxxxXxxx`, so `StyledValidationIcon`. Just a recommendation.

Reply:
Resolved in commit 18a6cb57f